docs/modules/ROOT/pages/documentation/payara-server/ejb/persistent-timers.adoc

```diff
@@ -62,3 +62,5 @@ asadmin> migrate-timers --target server1 server2
 ----
 
 Where `server1` is the active instance to migrate timers to and `server2` is the failed instance.
+
```

Comment:
Looks good, but due to how relevant and important the feature is, I'd put this statement in a separate section like this:

[[migration-live-instance]]
=== Migration from Live Instances

_Since Payara Server 5.2020.1_

The `migrate-timers` command can also be used to migrate timers that are scheduled to expire on a live instance. (WOULD BE GOOD TO ADD SOME INFO ON WHAT THIS MEANS AS WELL)

Reply:
Donezo. How's it looking now?